content/en/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -32,6 +32,13 @@ and when carrying out Pod eviction ranking.
 
 You need to make sure a `RuntimeClass` is utilized which defines the `overhead` field.
 
+## Understanding "kata-fc" 
```

Comment:
This shouldn't have it's own section. Can you integrate the explanation into the first paragraph of the example section instead? It shouldn't be more than a sentence either, as it's not really relevant to the example or the rest of the article.

Reply:
@tallclair Yeah sure, I will make those changes.